sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1133,9 +1124,22 @@ void foo(queue q /* queue in recording mode */ ) {
 }
 ----
 
-The copy will not occur if the buffer is accessed inside the submitted command
-group function with an accessor with `access_mode::write` or the `no_init`
-property.
```

Comment:
Would prefer to use Greg's suggested wording from the https://github.com/intel/llvm/pull/5626#discussion_r12315436000 thread


> The implementation guarantees that the host memory will not be copied internally if all the commands accessing this buffer use access_mode::write or the no_init property because the host memory is not needed in these cases. Note, however, that these cases require the application to disable copy-back as described in <<buffer-limitations>>.